### PR TITLE
Creating pages for publication review and ethics policies for submission to indices.

### PR DESCRIPTION
**Pull request by mrshirts**

### CODE REVIEW

File: _editorial_policies/04_publication_ethics.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+- An editor should respect the intellectual independence of authors.
+- Editorial responsibility and authority for any manuscript authored by an editor and submitted to the editor’s journal should be delegated to some other qualified person, such as another editor of that journal / member of its Editorial Advisory Board. Editors should also delegate manuscripts submitted by recent graduates, or by colleagues at the same institution, to some other qualified editor. A collaboration between the author and editor, which is separate from the paper under review, or may may not represent a conflict requiring manuscript delegation. However, an editor should not be responsible for a paper which would directly further their own research program; for example, an editor should not manage a paper that will serve as preliminary results for a grant or grant renewal that the editor would financially benefit from, even if they were not involved in writing the grant.  Editors should review the LiveCoMS bylaws for more information on what constitutes a conflict.
```
Comment (mrshirts) [inline _editorial_policies/04_publication_ethics.md:13]: This brought up a question if the conflicts is too lenient.  Specifically if a collaboration is automatically a conflict, and it defines more precisely what definitely would be a conflict.  The issue since so many editors are working on reviews, that we will run out of editors that can manage an article.  But since we have enough editors now, maybe this isn't a big deal to fix.

Maybe it should be changed to the same conflict for reviewers - a PUBLICATION isn't necessarily disqualifying if it's a large multiauthor publication that the two people didn't directly work on together.

File: _author_instructions/01_policies.md [hunk 2]
```diff
@@ -321,10 +325,10 @@ Domain Dedication (which can be found at
 https://creativecommons.org/publicdomain/zero/1.0/)."
 [See the analysis of this language here](https://theunitedstates.io/licensing/).
 
-## Prior Publication
+## Prior Publication and Plagarism
 
 Documents should not have been submitted in the current form to another journal, or be simultaneously under consideration for publication another journal.
-Preprints are not considered prior publication.
+Importatly, preprints are not considered prior publication.
```
Comment (davidlmobley) [inline _author_instructions/01_policies.md:331]: "Importantly"; but is this word needed here? I think it's important only if it's relevant to the person and is unlikely to need emphasis.

File: _author_instructions/01_policies.md [hunk 3]
```diff
@@ -333,6 +337,10 @@ Authors are responsible for obtaining all needed permissions to include previous
 If an article is an adaptation of a previously published article, it must be noted in the submission cover letter and major changes noted.
 Evaluating whether such changes constitute a significant revision will be part of the review process.
 
+The U.S. National Science Foundation definition of plagiarism as "the appropriation of another person’s ideas, processes, results, or words without giving appropriate credit" (45 Code of Federal Regulations, Section 689.1). Authors should not engage in plagiarism, which operationally means verbatim or near-verbatim copying, or even very close paraphrasing, of text or results from another’s work.
```
Comment (davidlmobley) [inline _author_instructions/01_policies.md:340]: "of plagiarism as" -> "of plagiarism is", or could be "defines plagiarism as..."

File: _author_instructions/01_policies.md [hunk 4]
```diff
@@ -333,6 +337,10 @@ Authors are responsible for obtaining all needed permissions to include previous
...
+
+Authors should not engage generally engage self-plagiarism, i.e. close replication of the authors' own previously published without acknowledgement of the source. Material quoted verbatim from the authors' own previously published work may be acceptable, as long as the overall work clearly notes that the work is partially based on identified previous work, and the broad outlines of the differences and similarities of the previous work are noted. This use of previous work only applies to material that could be reasonably considered review or pedagogcial; in no context should previously published results or methodology be depicted as new.  We emphasize again that authors themselves are responsible for obtaining all needed permissions to include previously published material, including figures.
```
Comment (davidlmobley) [inline _author_instructions/01_policies.md:342]: It seems like the exception to this paragraph is if someone has published previously under a permissive license that allows them to adapt their material for later reuse (e.g. with Ann. Rev., or under a CC-BY license elsewhere, or...) in which case they would simply need to acknowledge/cite the prior work and explain how they've adapted it, probably. 

Your statement gets close to saying something like this, but this part is confusing: "Material quoted verbatim from the authors' own previously published work may be acceptable, as long as the overall work clearly notes that the work is partially based on identified previous work, and the broad outlines of the differences and similarities of the previous work are noted." How can it be quoted verbatim if it's also similar and different? 

I think the correct policy should be something like:
- Any quotes from prior work are given as quotes, with proper citation/acknowledgement (and permissions), even if that prior work is from the authors
- Any adaptation from prior work (by the authors) is allowed only of the prior license permits, and the original source should be cited/acknowledged and the adaptation explained

File: _editorial_policies/04_publication_ethics.md (hunk 1 above)
Comment (davidlmobley) [reply _editorial_policies/04_publication_ethics.md:13]: I'm unclear on what you're saying here in your comment. Are you saying that this provides a more precise definition than our current one and is more helpful, so we should move to this? Or are you saying you want further refinement of this, and if so, how?

File: _editorial_policies/04_publication_ethics.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+- Unpublished information, arguments, or interpretations disclosed in a submitted manuscript should not be used in an editor’s own research except with the consent of the author. However, if such information indicates that some of the editor’s own research is unlikely to be profitable, the editor could ethically discontinue the work. When a manuscript is so closely related to the current or past research of an editor as to create a conflict of interest, the editor should arrange for some other qualified editor to take editorial responsibility for that manuscript. In some cases, it may be appropriate to tell an author about the editor’s research and plans in that area.
+- If an editor is presented with convincing evidence that the main substance or conclusions of a report published in an editor’s journal are erroneous, the editor should facilitate publication of an appropriate report pointing out the error.  This error should be corrected in the current GitHub version of the article, and a note may be placed on the original article indicating this correction.
```
Comment (davidlmobley) [inline _editorial_policies/04_publication_ethics.md:15]: I don't understand what situation this bullet point has in mind. Is it "Someone comes along proving that paper X is wrong, so the editor finds different authors to write a different paper showing that X is wrong?" Or is it "someone comes along proving that paper X is wrong, so the editor works with the authors of paper X to get the paper corrected or the error pointed out"? Or neitehr or both?

File: _editorial_policies/04_publication_ethics.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+- If an editor is presented with convincing evidence that the main substance or conclusions of a report published in an editor’s journal are erroneous, the editor should facilitate publication of an appropriate report pointing out the error.  This error should be corrected in the current GitHub version of the article, and a note may be placed on the original article indicating this correction. 
+- An author may request that the editor not use certain reviewers in consideration of a manuscript. However, the editor may decide to use one or more of these reviewers, if the editor feels their opinions are important in the fair consideration of a manuscript. This might be the case, for example, when a manuscript seriously disagrees with the previous work of a potential reviewer
```
Comment (davidlmobley) [inline _editorial_policies/04_publication_ethics.md:16]: I'm not sure, is this an important policy point to state? Basically it says that the editor can decide to do what they want; wouldn't this be the default position if not stated?

File: _editorial_policies/04_publication_ethics.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+- An author’s central obligation is to present an accurate and complete account of material presented, absolutely avoiding deception, including the data collected or used. In the case of presenting new data (which will generally not be the case in  LiveCoMS articles, but is not prohibited), the procedures, software links, parameters and the data provided should contain sufficient detail and reference to public sources of information to permit a trained professional to reproduce the observations. 
+- Authors should provide data and methods to other researchers, and are encouraged to submit their data to a public database, where available. 					
+- An author is obligated to cite those publications that have been influential in preparing their materials and that will guide the reader quickly to the earlier work that is essential for understanding the present investigation. 
+- In submitting a manuscript for publication, an author should inform the editor of related manuscripts that the author has under editorial consideration or in press. Copies of those manuscripts should be supplied to the editor, and the relationships of such manuscripts to the one submitted should be indicated.
```
Comment (davidlmobley) [inline _editorial_policies/04_publication_ethics.md:24]: Is this one relevant to us? We're not focused on original research.

File: _editorial_policies/04_publication_ethics.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+- Submitting authors must ensure that the listed co-authors of a paper should be all those persons who have made significant scientific contributions to the work reported and who share responsibility and accountability for the results. Authors should appropriately recognize the contributions of technical staff and data professionals. Other contributions should be indicated in a footnote or an “Acknowledgments” section. An administrative relationship to the work does not of itself qualify a person for co-authorship (but occasionally it may be appropriate to acknowledge major administrative assistance). Deceased persons who meet the criterion for inclusion as co-authors should be so included, with a footnote reporting date of death. No fictitious name should be listed as an author or coauthor. The author who submits a manuscript for publication accepts the responsibility of having included as co-authors all persons appropriate and none inappropriate. The submitting author should have sent each living co-author a draft copy of the manuscript and have obtained the co-author’s assent to co-authorship of it.
```
Comment (davidlmobley) [inline _editorial_policies/04_publication_ethics.md:29]: Don't we somewhere have a policy on author REMOVAL from subsequent versions of an article? I imagine in the case of a dead author they should likely be removed from subsequent versions if the authors list changes; an honorary authorship of new/distinct versions is probably not warranted.

File: _editorial_policies/04_publication_ethics.md [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+- Images should be free from misleading manipulation. An accurate description of how the images were generated and produced should be provided.  Images that are not original must have permission to be included.
```
Comment (davidlmobley) [inline _editorial_policies/04_publication_ethics.md:31]: "Images... must have permission" probably should be "Authors... must have permission... to include any images which are not original".

File: _editorial_policies/04_publication_ethics.md [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+  - In some special cases, the assigned reviewer may desire some level of input, advice, or guidance from a specific expert to facilitate review of the manuscript. In such instances, the reviewer must seek approval from the editor prior to sharing the contents of the manuscript with the expert. Following editorial approval, the assigned reviewer must clearly communicate the need for confidentiality to the consulted expert. The identity of the expert must also be disclosed to the editor at that time.  
+  - In some special cases, the reviewer may wish to disclose his/her identity to the manuscript authors, the editor should be consulted. There may be instances where the editor requests that the reviewer retains anonymity. The decision to disclose such information to authors is the sole discretion of the editor. However, such decisions should be accompanied by appropriate rationale from the editor.
```
Comment (davidlmobley) [inline _editorial_policies/04_publication_ethics.md:44]: "the editor should be consulted" is not grammatically correct as written; probably you mean "authors; in such cases, the editor should be consulted first," or similar. But do we really want this? There are some reviewers - Jan Jensen comes to mind - who ALWAYS want to do "onymous" reviews. I have no problem with this and don't see why they should need permission first. Anonymous review provides protection for the reviewer; if they don't want that protection it seems fine to me.

File: _editorial_policies/04_publication_ethics.md [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+  - In some special cases, the assigned reviewer may desire some level of input, advice, or guidance from a specific expert to facilitate review of the manuscript. In such instances, the reviewer must seek approval from the editor prior to sharing the contents of the manuscript with the expert. Following editorial approval, the assigned reviewer must clearly communicate the need for confidentiality to the consulted expert. The identity of the expert must also be disclosed to the editor at that time.  
+  - In some special cases, the reviewer may wish to disclose his/her identity to the manuscript authors, the editor should be consulted. There may be instances where the editor requests that the reviewer retains anonymity. The decision to disclose such information to authors is the sole discretion of the editor. However, such decisions should be accompanied by appropriate rationale from the editor. 
+  - Reviewers may disclose publicly that they have served as a reviewer for LiveCoMS. However, under no circumstances should the reviewer identify himself/herself as the reviewer of a specific manuscript.
```
Comment (davidlmobley) [inline _editorial_policies/04_publication_ethics.md:45]: Same comment as above. Also, this would preclude anyone doing open commenting on GitHub in ways that could reveal they had reviewed the paper, whereas I think this is something we'd like to see happen!

File: _editorial_policies/04_publication_ethics.md [hunk 12]
```diff
@@ -0,0 +1,51 @@
...
+- A reviewer should act promptly, submitting a report in a timely manner. Should a reviewer receive a manuscript at a time when circumstances preclude prompt attention to it, the unreviewed manuscript should be returned immediately to the editor. Alternatively, the reviewer might notify the editor of probable delays and propose a revised review date.
+- Reviewers should not use or disclose unpublished information, arguments, or interpretations contained in a manuscript under consideration, except with the consent of the author. If this information indicates that some of the reviewer’s work is unlikely to be profitable, the reviewer, however, could ethically discontinue the work. In some cases, it may be appropriate for the reviewer to write the author, with copy to the editor, about the reviewer’s research and plans in that area.
```
Comment (davidlmobley) [inline _editorial_policies/04_publication_ethics.md:49]: This point doesn't really apply since everything should be publicly up on GitHub before it's being reviewed. So the reviewer should be able to talk about it publicly with anyone, I think.

File: _editorial_policies/04_publication_ethics.md [hunk 13]
```diff
@@ -0,0 +1,51 @@
...
+- The review of a submitted manuscript may sometimes justify criticism, even severe criticism, from a reviewer. However, in no case is personal criticism of the author considered to be appropriate. 
+- Reviewers should notify editors of concerns with respect to manuscripts that report research that, based on current understanding, can be reasonably expected to provide knowledge, products, or technologies that could be directly misapplied by others to pose a threat to public health and safety, agricultural crops and other plants, animals, the environment, or materiel.
```
Comment (davidlmobley) [inline _editorial_policies/04_publication_ethics.md:51]: I don't think this last one applies to us either?

**Review by davidlmobley — CHANGES_REQUESTED** [review]
Left some comments; mostly looks good but some may need fixes or refinement to fit our circumstances.

File: _editorial_policies/04_publication_ethics.md (hunk 1 above)
Comment (mrshirts) [reply _editorial_policies/04_publication_ethics.md:13]: @dmzuckerman originally said that "collaborator" should automatically be a conflict, I was trying to say it might be a conflict.

File: _editorial_policies/04_publication_ethics.md (hunk 5 above)
Comment (mrshirts) [reply _editorial_policies/04_publication_ethics.md:15]: Maybe it's just that they should ask the authors to correct it on the GitHub repository, and any future revisions would be accepted conditional on the concerns being resolved?

Comment (mrshirts) [reply _editorial_policies/04_publication_ethics.md:15]: Added text to that effect.

File: _editorial_policies/04_publication_ethics.md (hunk 6 above)
Comment (mrshirts) [reply _editorial_policies/04_publication_ethics.md:16]: It  was in the ACS version - I don't  see a harm in making this explicit.

File: _editorial_policies/04_publication_ethics.md (hunk 8 above)
Comment (mrshirts) [reply _editorial_policies/04_publication_ethics.md:29]: I added a comment about authorship with living documents and a link to the relevant text.

File: _editorial_policies/04_publication_ethics.md (hunk 9 above)
Comment (mrshirts) [reply _editorial_policies/04_publication_ethics.md:31]: yeah, simpler.

File: _editorial_policies/04_publication_ethics.md (hunk 10 above)
Comment (mrshirts) [reply _editorial_policies/04_publication_ethics.md:44]: Good point, I have make this more consistent with the rest of the LiveCoMS policies.

File: _editorial_policies/04_publication_ethics.md (hunk 12 above)
Comment (mrshirts) [reply _editorial_policies/04_publication_ethics.md:49]: good point, eliminated.

File: _editorial_policies/04_publication_ethics.md (hunk 13 above)
Comment (mrshirts) [reply _editorial_policies/04_publication_ethics.md:51]: Yeah, since it is all public, I think so.  It's also unlikely that there is any such research (since most of the material is review), but might as well.

File: _editorial_policies/04_publication_ethics.md (hunk 7 above)
Comment (mrshirts) [reply _editorial_policies/04_publication_ethics.md:24]: Well, I think it's relevant if there are overlaps in material covered.

File: _author_instructions/01_policies.md (hunk 4 above)
Comment (mrshirts) [reply _author_instructions/01_policies.md]: OK, made changes to that effect.

File: _editorial_policies/04_publication_ethics.md (hunk 1 above)
Comment (davidlmobley) [reply _editorial_policies/04_publication_ethics.md:13]: OK. Suggest "or may may not" -> "may or may not"

File: _editorial_policies/04_publication_ethics.md (hunk 7 above)
Comment (davidlmobley) [reply _editorial_policies/04_publication_ethics.md:24]: It increases the burden on editors; if I have to just assign a manuscript for review, I can do it promptly. If I have to read the manuscript and two or three other related manuscripts to decide whether they are related and how much, that might take me a couple of weeks to get to. And that's even before assigning reviewers. Then I have to decide if this material should also go to the reviewers and if I should get them to assess it. I see it lengthening the review process considerably, and we haven't given any guidance on how this would factor in to the editorial decision/review process. It will likely bubble up to the managing editors and we'll have to debate how we should handle it. If that's what we want that's fine, but I just don't think it should be needed.

Comment (mrshirts) [reply _editorial_policies/04_publication_ethics.md:24]: How about just inform the editors? Then we can discuss it at a higher level, without worry about the details.
